src/Libraries/Watch3DNodeModels/Watch3D.cs

```diff
@@ -310,5 +286,23 @@ private void DeserializeCamera()
             // No visualization update is required for this node type.
             return false;
         }
+
+        protected override string GetInputTooltip(int index)
+        {
+            return string.Empty;
```

Comment:
we don't want tooltips or names?

Reply:
I just added tooltip. Keeping the same as the current tooltip "Incoming Geometry Objects". I think no need for name as the default